src/dos/program_imgmount.cpp

```diff
@@ -36,12 +36,62 @@
 #include "string_utils.h"
 #include "../ints/int10.h"
 
+void IMGMOUNT::ListImgMounts(void)
+{
+	const std::string header_drive = MSG_Get("PROGRAM_MOUNT_STATUS_DRIVE");
+	const std::string header_name = MSG_Get("PROGRAM_MOUNT_STATUS_NAME");
+	const std::string header_label = MSG_Get("PROGRAM_MOUNT_STATUS_LABEL");
+	const std::string header_slot = MSG_Get("PROGRAM_MOUNT_STATUS_SLOT");
+
+	const int term_width = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
+	const auto width_1 = static_cast<int>(header_drive.size());
+	const auto width_3 = std::max(minimum_length,
+	                              static_cast<int>(header_label.size()));
+	const auto width_4 = std::max(minimum_length,
+	                              static_cast<int>(header_slot.size()));
+	const auto width_2 = term_width - 4 - width_1 - width_3 - width_4;
+	if (width_2 < 0) {
+		LOG_WARNING("Message is too long.");
+		return;
+	}
+
+	auto print_row = [&](const std::string &txt_1, const std::string &txt_2,
+	                     const std::string &txt_3, const std::string &txt_4) {
+		WriteOut("%-*s %-*s %-*s %-*s\n", width_1, txt_1.c_str(),
+		         width_2, txt_2.c_str(), width_3, txt_3.c_str(),
+		         width_4, txt_4.c_str());
+	};
```

Comment:
I'm not sure if these colums have any different meanings from each other?
when I see `txt_1`, `txt_2`, `txt_3`, ... it tells me we're dealing with ambiguous "bulk data".

But if these columns have some meaning, lets hold the data in variables named after those meanings.

Reply:
Changed such names to more descriptive ones for both IMGMOUNT and MOUNT commands. Done.